pypdf/_utils.py

```diff
@@ -190,6 +190,23 @@ def skip_over_whitespace(stream: StreamType) -> bool:
     return cnt > 1
 
 
+def check_if_whitespace_only(value: bytes) -> bool:
+    """
+    Check if the given value consists of whitespace characters only.
+
+    Args:
+        value: The bytes to check.
+
+    Returns:
+        True if the value only has whitespace characters, otherwise return False.
+    """
+    for index in range(len(value)):
+        current = value[index:index + 1]
+        if current not in WHITESPACES:
+            return False
+    return True
```

Comment:
What do you think about this:

```suggestion
def is_whitespace_only(value: bytes) -> bool:
    """
    Check if the given value consists of whitespace characters only.

    Args:
        value: The bytes to check.

    Returns:
        True if the value only has whitespace characters, otherwise return False.
    """
    for char in value:
        if char not in WHITESPACES:
            return False
    return True
```

Reply:
This will not work - iterating over `bytes` will lead to their integer values.

Reply:
Interesting - thanks for clarifying!

Aparently there might be a better way in Python 3.12 ... so with the current speed, we will be able to use that in 10 years xD

https://stackoverflow.com/a/14267935/562769